resources/options.js

```diff
@@ -153,11 +156,27 @@ function switchOptions() {
     }
 }
 
+// Saves the default options by saving an object.
+function saveDefaults() {
+    for (var i = 0; i < options.length; i++) {
+        defaultOptions[options[i]] = optionsElement(i).prop('checked');
+    }
+}
+
+// Sets the options to the default options by updating the chrome storage and the options page.
+function restoreDefaults() {
+    chrome.storage.sync.set(defaultOptions, function() {});
```

Comment:
I don't think it is really necessary either. The only thing that could be confusing is when the user clicks the button when the default settings are already set, then the user wouldn't get any visual feedback